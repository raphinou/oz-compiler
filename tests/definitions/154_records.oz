% Records where values are results of function call
% --SKIP TEST--
local
   R
   fun{F I}
      I*2
   end
in
   A=1
   B=2
   R=rec(a:A b:B)
   R2=rec(a r:R f:{F 2})
   {Show R2}
end

