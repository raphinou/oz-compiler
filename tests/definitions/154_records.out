rec(1:a f:4 r:rec(a:1 b:2))
